Reflector/Reflector.tests/ReflectorTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Reflector.tests;
```

Comment:
```suggestion
﻿namespace Reflector.Tests;
```
Неймспейсы в .NET именуются с заглавной всегда, соответственно и проекты